book/website/community-handbook/community-calls/community-calls-coworking-dropins.md

```diff
@@ -1,14 +1,9 @@
 (ch-coworking-dropins)=
 # Drop-in Sessions
```

Comment:
Reading the materials - I realised I got a bit confused, is the other page on coworking completely deleted?

Reply:
Coworking instructions is Part of collaboration guide, it doesn't have to be duplicated here, which was the case (I had repeated that but seems completely unnecessary when I reviewed this chapter). I have cross referenced to the chapter in collaboration guide.

Reply:
Coworking instructions is Part of collaboration guide, it doesn't have to be duplicated here, which was the case (I had repeated that but seems completely unnecessary when I reviewed this chapter). I have cross referenced to the chapter in collaboration guide.

Reply:
Here is how it is displayed in the book: 
<img width="326" alt="image" src="https://github.com/user-attachments/assets/f192b417-a5a0-4030-8bfc-de51283fd43b" />